src/owner/handle.rs

```diff
@@ -28,9 +28,9 @@ pub async fn handle_message(
     }
 
     // match second token to a command, if any
-    match tokens[1] {
-        "sql" => commands::sql::run_sql(bot, event).await?,
-        "embed" => commands::embed_test::test_starboard_embed(bot, event).await?,
+    match tokens.get(1) {
+        Some(&"sql") => commands::sql::run_sql(bot, event).await?,
+        Some(&"embed") => commands::embed_test::test_starboard_embed(bot, event).await?,
```

Comment:
pointless change, it returns if tokens.len() < 2

Reply:
mb didn't notice